<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>311</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="addtab">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>enter the name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>enter Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPhoneNumber">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PhoneNumber :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>enter number</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="addbtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>AddContact</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblEmailAddress">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EmailAddress :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="removetab">
     <attribute name="title">
      <string>Remove</string>
     </attribute>
     <widget class="QLabel" name="zz">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Index : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PhoneNumber:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>enter the Index</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter the PhoneNumber</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rmvbtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchtab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="searchbtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>353</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue;</string>
    </property>
    <property name="text">
     <string>index</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchidx">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>353</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPhonebook">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuPhonebook"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
